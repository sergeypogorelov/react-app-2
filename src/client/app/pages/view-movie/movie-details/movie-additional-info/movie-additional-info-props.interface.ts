export interface MovieAdditionalInfoProps {
  value: string;
  unit: string;
}
