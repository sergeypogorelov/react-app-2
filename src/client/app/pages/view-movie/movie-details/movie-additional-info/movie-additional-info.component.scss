@import '../../../../../styles/variables';

.movie-additional-info {
  &-item {
    display: flex-end;
  }
  &-value {
    margin-right: 5px;
    font-size: 26px;
    color: $c-primary;
  }
  &-unit {
    font-size: 14px;
    color: rgba($color: $c-default, $alpha: 0.5);
  }
}
