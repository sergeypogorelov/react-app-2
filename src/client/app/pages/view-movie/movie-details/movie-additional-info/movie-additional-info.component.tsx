import './movie-additional-info.component.scss';

import React, { FunctionComponent } from 'react';

import { MovieAdditionalInfoProps } from './movie-additional-info-props.interface';

export const MovieAdditionalInfo: FunctionComponent<MovieAdditionalInfoProps> = ({
  value,
  unit,
}) => {
  return (
    <div className="movie-additional-info">
      <span className="movie-additional-info-value">{value}</span>
      <span className="movie-additional-info-unit">{unit}</span>
    </div>
  );
};
