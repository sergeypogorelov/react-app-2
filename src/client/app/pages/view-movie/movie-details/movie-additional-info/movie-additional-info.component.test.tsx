import React from 'react';
import renderer from 'react-test-renderer';

import { MovieAdditionalInfo } from './movie-additional-info.component';
import { MovieAdditionalInfoProps } from './movie-additional-info-props.interface';

test('MovieAdditionalInfo component should render correctly', () => {
  const props: MovieAdditionalInfoProps = {
    value: 'value',
    unit: 'unit',
  };
  const tree = renderer.create(<MovieAdditionalInfo {...props} />).toJSON();
  expect(tree).toMatchSnapshot();
});
