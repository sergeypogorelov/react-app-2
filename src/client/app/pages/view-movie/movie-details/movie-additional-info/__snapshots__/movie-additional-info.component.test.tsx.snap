// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieAdditionalInfo component should render correctly 1`] = `
<div
  className="movie-additional-info"
>
  <span
    className="movie-additional-info-value"
  >
    value
  </span>
  <span
    className="movie-additional-info-unit"
  >
    unit
  </span>
</div>
`;
